### ISSUE TITLE
app hangs on Run Android on Simulators

### ISSUE THREAD

**Issue by bochen2014** [Microsoft/vscode-react-native#538]
according to documents, the 'run android (on simulator or devices)'   command will:
### The Run Android command triggers `react-native run-android` and starts your app for Android.

but I have noticed that it does more than that. 
the default `react-native run-android` uses Chrome as its default debugger tool, while the `run android` command in this tool will register `vscode` as the default remote debugger tool. 
Therefore, if you run `Run Android` without starting a debug session , you app will first hang if it has 'remote debug on' and eventually, get a red box error saying "Unable to connect with remote debugger"
(because it's trying to connect to `vscode` debugger port , which is not open yet)

not sure if this is intended or not. I have to click `stop remote debugging` if I just want to run `Run Android on simulator` command

**Comment by bochen2014** [comment]
![image](https://user-images.githubusercontent.com/8207081/30779666-1f00b96c-a13a-11e7-9a61-48d44b909270.png)

disable remote debugging will solve the issue.
It seems to me that, I can't use Chrome debug and `Run Android on Simulator` together. Am I correct?

**Comment by iTOYS** [comment]
Hello, @bochen2014!
If i right understood you.
For debugging app with `Chrome` please:
1. Run `Run Android on Simulator` command from vscode command palette  
2. Open [http://localhost:8081/debugger-ui](http://localhost:8081/debugger-ui) in chrome
If [`Debug JS Remotely`](http://facebook.github.io/react-native/docs/debugging.html) enabled on application, debugger will be attached.

Thanks!

**Comment by bochen2014** [comment]
right....

I think I figured out what happened before. 
previously, I ran `Debug Android` first, and once my app was loaded into simulator, I clicked  'Debug JS Remotely' ; note that after this, the application will connect to vscode debugger port (can be configured via 'internalDebuggerPort in launch.json; otherwise `vscode` will randomly allocate one) for remote debugging;
After this, I was exploring `Run Android on Simulators` on *the same simluator*, note that 
1. vscode debug port is not opened this time
2. due to *cache* issue, the application is 
    a. has 'remote debugging' on
    b. has remote debugger set to `vscode`
therefore, the application hangs. 

to fix the issue, after run `Run Android on Simluators` we need to 
1. disable remote debugging
2. reload
3. enable remote deubgging
this is force application to connect to Chrome debugger (#8081)

Hope it will help somebody
